corehq/apps/hqadmin/forms.py

```diff
@@ -117,6 +95,75 @@ def __init__(self, can_toggle_is_staff, *args, **kwargs):
         )
 
 
+class OffboardingUserListForm(forms.Form):
+    csv_email_list = forms.CharField(
+        label="Comma seperated email addresses",
+        widget=forms.Textarea()
+    )
```

Comment:
given that you can now get the inverse of the match, I think this field should likely be considered optional, so that, combined with the inverse field, you can submit no addresses and retrieve all staff members.  Doing so would require changes to the clean logic to support an empty field, however

Reply:
I don't think this is necessary, since you're already presented with a list of all staff members from the start. Unless you meant getting that list back after filtering it to a shorter list but it'd still be easier just to click the `Get users for offboarding` tab on the side.